drivers/CoreSPI/include/CoreSPI.h

```diff
@@ -0,0 +1,33 @@
+// Leka - LekaOS
+// Copyright 2021 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef _LEKA_OS_DRIVER_CORE_SPI_H_
+#define _LEKA_OS_DRIVER_CORE_SPI_H_
+
+#include "drivers/SPI.h"
+
+#include "interface/drivers/SPI.h"
+
+namespace leka {
+
+class CoreSPI : public interface::SPI
+{
+  public:
+	explicit CoreSPI(mbed::SPI &spi) : _spi {spi} {};
+
+	auto write(int value) -> int override;
+
+	auto write(const uint8_t *tx_buffer, int tx_length, uint8_t *rx_buffer, int rx_length) -> int override;
```

Comment:
même si `size_t == int`, ça rend l'interface plus clair de l'utiliser car on voit dans le nom du type qu'on attend une taille.

```suggestion
	auto write(const uint8_t *tx_buffer, size_t tx_length, uint8_t *rx_buffer, size_t rx_length) -> int override;
```